pfb-analysis/import/import_jobs.sh

```diff
@@ -38,8 +38,12 @@ function import_job_data() {
     NB_DATA_TYPE="${2-main}"    # Either 'main' or 'aux'
     NB_JOB_FILENAME="${NB_STATE_ABBREV}_od_${NB_DATA_TYPE}_JT00_2014.csv"
 
-    wget -P "${NB_TEMPDIR}" "http://lehd.ces.census.gov/data/lodes/LODES7/${NB_STATE_ABBREV}/od/${NB_JOB_FILENAME}.gz"
-    gunzip -c "${NB_TEMPDIR}/${NB_JOB_FILENAME}.gz" > "${NB_TEMPDIR}/${NB_JOB_FILENAME}"
+    if [[ -f "../../data/${NB_JOB_FILENAME}.gz" ]]; then
+        gunzip -c "../../data/${NB_JOB_FILENAME}.gz" > "${NB_TEMPDIR}/${NB_JOB_FILENAME}"
```

Comment:
Since this is running in the container, which is mounted to `/data`, maybe the relative dir prefixes go away and we just check `/data/${NB_JOB_FILENAME}`?